llvm/docs/LangRef.rst

```diff
@@ -1634,6 +1634,24 @@ Currently, only the following parameter attributes are defined:
 
     This attribute cannot be applied to return values.
 
+``range(<ty>,<a>,<b>)``
+    This attribute expresses the possible range the parameter is in. If the 
+    value of the parameter is not in the specified range, a poison value is 
+    returned instead.
```

Comment:
I don't like "is returned", this is not a function.

Reply:
it was copied from the range metadata but I agree it sounds strange, how about:
The value of the parameter is in the specified range or is poison.